### PR TITLE
Phase2 tracker geometries: Add 20% and 50% material variation 

### PR DESCRIPTION
**Pull request by mdhildreth**
Add two geometry scenarios with increased material for L1 Track Trigger Stress Test.

### CODE REVIEW

**Comment by mdhildreth** [comment]
please test

**Comment by mdhildreth** [comment]
please test

**Comment by mdhildreth** [comment]
@fabiocos @kpedro88 I can update the numbering here to something larger than D25 so this doesn't conflict with the MTD PR.

**Comment by kpedro88** [comment]
@mdhildreth yes, please change it to 26/27, thanks

**Comment by mdhildreth** [comment]
@fabiocos : I updated the numbering of the geometries so that this set will not conflict with your MTD PR.  You should merge yours first, then I can merge on top of it.

**Comment by mdhildreth** [comment]
please test

**Comment by ianna** [comment]
+1

**Comment by kpedro88** [comment]
I am holding off on signing this for the moment to see if #23126 converges soon... it is easier to rebase this one than that one.

**Comment by fabiocos** [comment]
@kpedro88 I am testing now #23126 the pending issues should be solved. Otherwise I can strip the geometry and SIM parts, which lokk ok, and move forward with those so as not keep this PR on hold for a long time

**Comment by kpedro88** [comment]
@mdhildreth this can be rebased on the next IB. I would recommend squashing all the commits into one commit before rebasing to avoid repeated conflicts in the same files.

**Comment by davidlange6** [comment]
Your PR is unmergeable. Please have a look and possibly rebase it.

**Comment by mdhildreth** [comment]
@fabiocos I can rebase this and have it ready very quickly

**Comment by fabiocos** [comment]
@mdhildreth please try to get it ready before tonight's build, unfortunately we had several overlapping developments conflicting among themselves. I would like to avoid to move pre4 to the week-end

**Comment by mdhildreth** [comment]
@fabiocos I'll have it done in about 20min.  Thanks.

**Comment by fabiocos** [comment]
@mdhildreth for my info, what are the plans of use for this PR in pre4? Special relval samples for dedicated studies? Or is it just a basis for further developments?

**Comment by mdhildreth** [comment]
@fabiocos  Special Relvals for the Track Trigger stress test (which was supposed to be done by Tracker Week two weeks ago...)

File: Configuration/StandardSequences/python/GeometryConf.py [hunk 1]
```diff
@@ -35,4 +35,7 @@
     'Extended2023D23' : 'Extended2023D23,Extended2023D23Reco',
     'Extended2023D24' : 'Extended2023D24,Extended2023D24Reco',
     'Extended2023D25' : 'Extended2023D25,Extended2023D25Reco'
+    'Extended2023D25' : 'Extended2023D25,Extended2023D25Reco',
```
Comment (fabiocos) [inline Configuration/StandardSequences/python/GeometryConf.py]: this is a duplication to be removed, right?

**Comment by mdhildreth** [comment]
@fabiocos Yes, just committed.  Got interrupted here... Rebased on top of latest IB

**Comment by fabiocos** [comment]
@mdhildreth I believe that the duplicated line is still there (Extended2023D25)

**Comment by fabiocos** [comment]
@mdhildreth unfortunately the additional tests cannot be verified here, but I assume this has been done in a standalone way

**Comment by fabiocos** [comment]
please test

**Comment by mdhildreth** [comment]
@fabiocos even with that fixed, the geometry tests fail. I'm trying to figure out why.

**Comment by fabiocos** [comment]
20:48 cmsdev15 2238> $CMSSW_BASE/src/Configuration/Geometry/test/runtests.sh 
GeometryExtended2023D25_cff.py differs
GeometryExtended2023D25Reco_cff.py differs
GeometryExtended2023D24_cff.py differs
GeometryExtended2023D24Reco_cff.py differs

**Comment by mdhildreth** [comment]
@fabiocos  ok, will do.

**Comment by fabiocos** [comment]
please abort test

**Comment by mdhildreth** [comment]
please test

**Comment by mdhildreth** [comment]
@fabiocos ok, that should do it.

**Comment by fabiocos** [comment]
@ianna I think there is not relevant change wrt your signature
@fabozzi @prebello the PyRelVal part looks straightforward
@kpedro88 I understood you was ok with this 

Please check and sign in case, the new scenarios cannot really be tested by the regular short matrix (I executed one D26 test workflow standalone)

**Comment by kpedro88** [comment]
+1

**Comment by fabiocos** [comment]
@ianna @fabozzi @prebello could you please check and sign in case?

**Comment by fabiocos** [comment]
+operations

the update of StandardSequences is just an addition of new 2023 geometry scenarios

**Comment by ianna** [comment]
+1

**Comment by fabiocos** [comment]
+1

the PyReleaseValidation update looks consistent with previous updates of upgrade scenarios

**Comment by fabiocos** [comment]
merge

**Comment by fabozzi** [comment]
+1

**Pull request merged**